br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -458,7 +458,7 @@ func setDatumByString(d *types.Datum, v string, meta *parquet.SchemaElement) {
 		ts = ts.UTC()
 		v = ts.Format(utcTimeLayout)
 	}
-	d.SetString(v, "")
+	d.SetString(v, "utf8mb4_bin")
```

Comment:
There are many places need to consider string encodings, one is string data in parquet file, the other one is string variables in the memory of lightning process which read by parquet reader. Since golang string is always  assumed utf8-encoded I think this PR is OK. But I'm not sure if parquet file has another encoding for string data and go-parquet reader wrongly cast it to golang string without encode/decode.